### PR TITLE
More lint rules

### PR DESCRIPTION
**Pull request by domoritz**

### CODE REVIEW

File: src/compiler/facet.ts [hunk 1]
```diff
@@ -90,7 +90,7 @@ export function facetMixins(model: Model, marks) {
     // TODO: add properties to make rule optional
     rootMarks.push(getColumnRulesGroup(model, cellWidth));
   } else { // doesn't have column
-    if (model.has(Y)) { //keep y axis in the cell
+    if (model.has(Y)) { // keep y axis in the cell
```
Comment (kanitw) [inline src/compiler/facet.ts]: This rules looks really annoying unless there is a auto-fixer.

Comment (kanitw) [inline src/compiler/facet.ts]: I mean it's a good thing but I don't think it is very important.

Comment (domoritz) [inline src/compiler/facet.ts]: It's not but there were only 3 cases where this happened. I'm happy to
remove the rule.

File: src/compiler/marks.ts [hunk 2]
```diff
@@ -31,7 +31,7 @@ export function compileMarks(model: Model): any[] {
       type: MARKTYPES_MAP[mark],
       from: {
         // from.data might be added later for non-facet, single group line/area
-        transform: [{type: 'sort', by: sortBy}]
+        transform: [{ type: 'sort', by: sortBy }]
```
Comment (kanitw) [inline src/compiler/marks.ts]: This rules is nice if computer automatically do for us, again if there is no nice way to fix this without screwing up other style (Running formatter for the whole file is generally risky), let's turn this off.

File: src/compiler/facet.ts (hunk 1 above)
Comment (kanitw) [inline src/compiler/facet.ts]: Thinking more about this -- keep it then.

File: src/compiler/marks.ts (hunk 2 above)
Comment (kanitw) [inline src/compiler/marks.ts]: (This one is more annoying than the other one) -- no need to revert the change you already made -- it looks nicer but I just don't want to enforce it.  :)

Comment (domoritz) [inline src/compiler/marks.ts]: I did this automatically. I just right clicked and said "beautify content". It's not enforced, I believe and I definitely don't want this enforced.

Comment (kanitw) [inline src/compiler/marks.ts]: Sounds good then!   

Btw, do you mean Atom's "Format Code"?

Comment (domoritz) [inline src/compiler/marks.ts]: screen shot 2015-12-04 at 14 54 34
